src/mame/video/namco_c169roz.cpp

```diff
@@ -151,8 +202,7 @@ void namco_c169roz_device::unpack_params(const uint16_t *source, roz_parameters
 
 void namco_c169roz_device::draw_helper(screen_device &screen, bitmap_ind16 &bitmap, tilemap_t &tmap, const rectangle &clip, const roz_parameters &params)
 {
-	if (!m_is_namcofl)
-//  if (m_gametype != NAMCOFL_FINAL_LAP_R) // Fix speedrcr some title animations, but broke at road scene
+	if (!m_disable_custom_draw)
```

Comment:
Not sure I like the definition of “custom draw”, what exactly is supposed to be?